daisy_workflows/image_import/import_image.sh

```diff
@@ -84,6 +83,21 @@ function resizeDisk() {
   exit
 }
 
+# Determines whether a file is an OVA archive.
+# Arguments:
+#   File to check.
+# Returns:
+#   0 if the file is a tar file, non-zero otherwise.
+function fileIsOva() {
+  if [[ $(file --mime-type "$1") =~ application/x-tar ]]; then
+    contents=$(tar --list -f "$1")
+    if [[ "$contents" =~ ovf && "$contents" =~ vmdk ]]; then
```

Comment:
OVAs can have disks in any format, not only vmdk

Reply:
That piece of legacy code was built on an incorrect assumption that OVAs contain disks only in VMDK format. We should fix this eventually. Can you please create a tracking bug for this fix?